components/supervisor/pkg/supervisor/services.go

```diff
@@ -933,5 +934,6 @@ func (s *portService) RetryAutoExpose(ctx context.Context, req *api.RetryAutoExp
 
 // ResourcesStatus provides workspace resources status information.
 func (s *statusService) ResourcesStatus(ctx context.Context, in *api.ResourcesStatuRequest) (*api.ResourcesStatusResponse, error) {
-	return Top(ctx)
+	var data *api.ResourcesStatusResponse = s.topService.data
+	return data, nil
```

Comment:
(nit) `return s.topService.data, nil` - top service should ensure consistency of data, clients don't need to worry

Reply:
I've updated the `return` (removed the unnecessary pointer), but I don't understand if you are suggesting any more change by saying: " top service should ensure consistency of data, clients don't need to worry" - I agree with it, but is there any actionable on it?

Unless you're regerring to the change to the `return` but I think that is purely aesthetic change 🤔

Reply:
ok, i though we initially introduced it to create a new pointer and avoid a situation when top service updates it later, and it was left over